fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.fcrepo.auth.common;
 
-import static org.fcrepo.kernel.api.FedoraJcrTypes.JCR_CONTENT;
+import static org.modeshape.jcr.api.JcrConstants.JCR_CONTENT;
```

Comment:
`fcrepo-jms` has a test-dependency, which doesn't matter. It would be expected that `fcrepo-webapp` depends on the modeshape impl. Both `fcrepo-connector-file` and `fcrepo-auth-common` have a special relationship to modeshape and should remain as they are. The only concerning dependencies are `fcrepo-http-api` and `fcrepo-http-commons`.